src/test/rpc/GRPCTestClientBase.h

```diff
@@ -20,6 +20,7 @@
 #ifndef RIPPLED_GRPCTESTCLIENTBASE_H
 #define RIPPLED_GRPCTESTCLIENTBASE_H
 
+#include <grpcpp/grpcpp.h>
```

Comment:
I don't think this `#include` is necessary.

Reply:
removed